src/Profiler/NodeVisitor/ProfilerNodeVisitor.php

```diff
@@ -66,7 +67,7 @@ protected function doLeaveNode(Node $node, Environment $env)
 
     private function getVarName(): string
     {
-        return sprintf('__internal_%s', hash('sha256', $this->extensionName));
+        return sprintf('__internal_profiler_%d', $this->varNameSalt++);
```

Comment:
this now returns different variable names for each call to that method for the same visitor, and won't return different names for different visitors used by different extensions (the Symfony WebProfiler integration relies on that visitor too IIRC).
So this change looks wrong to me.

Reply:
You are right. I moved the `varName` to a variable, computed only once.